app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -9,23 +9,12 @@
 .gem-c-cards__list {
   list-style: none;
   padding: 0;
-  // Remove the outermost left and right margin from the internal margin of the list items.
-  margin: 0 govuk-spacing(-3);
-
+  margin: 0;
   display: grid;
-  grid-auto-flow: row;   // Use CSS grid to calculate the number of rows
-  grid-auto-rows: fractions(1); // Set all rows to same fractional height of the complete grid
-  grid-template-columns: fractions(1);
-
-  @include govuk-media-query($from: "tablet") {
-    grid-template-columns: fractions(2);
-  }
+  grid-column-gap: govuk-spacing(6); // Use deprecated grid-column-gap for Grade C browser support
+  column-gap: govuk-spacing(6);
 
   @include govuk-media-query($from: "desktop") {
-    // Note that browsers that don't support CSS grid display the component as one column on all
-    // breakpoints
-    grid-template-columns: fractions(3);
-
     // For browsers that don't support CSS grid, constrain the width to 50% to improve usability
     // especially for screen magnifier users
     width: 50%;
```

Comment:
Question - if non-grid browsers restrict the single column default component width to 50% for accessibility, shouldn't that be the default? If someone's using a screen magnifier on a modern browser, they'd get the full width, which is implied by the comment to not be as accessible?

This might be a bigger question than this PR, so maybe not worth agonising over for the initial release.

Reply:
Good point, definitely worth thinking about further and how we provide support for these browsers in the future.

When using the one column layout variation on desktop in browsers that do support CSS grid, the content of the card will be restricted to two-thirds (66.667%) for the [heading](https://github.com/alphagov/govuk_publishing_components/pull/4118/files#diff-0f64daead1e205c757c525d6c2800f6716edee2c7046bc590883efc2cf42ef92R59-R61) and [text](https://github.com/alphagov/govuk_publishing_components/pull/4118/files#diff-0f64daead1e205c757c525d6c2800f6716edee2c7046bc590883efc2cf42ef92R109-R112). Although not exactly the same implementation, we are quite close to the 50% width referenced here.